docs/design/messaging/message-round-trip.puml

```diff
@@ -0,0 +1,52 @@
+@startuml
```

Comment:
png up there is empty btw

Reply:
It have been lost somewhere during rebase commit split actions. I have regenerated it from puml source.